### PR TITLE
Milestone 1: Add App homepage

### PR DESCRIPTION
**Pull request by Felix45**
## Hello Microverse App: Homepage
In this milestone I:
- Added a homepage for the Hello Microverse App
- Added CSS Styles for the Hompage

### CODE REVIEW

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Felix45👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Repository looks professional.✔️
- Github flow was correctly used.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+name: Linters
```
Comment (brytebee) [inline .github/workflows/linters.yml:1]: - [ ] **Required:** Kindly replace this linter files with the appropriate one [here.](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml)

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Felix45👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- Repository looks professional.✔️
- Github flow was correctly used.✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .github/workflows/linters.yml (hunk 1 above)
Comment (Felix45) [reply .github/workflows/linters.yml:1]: Hi, @brytebee 

This has been well received and acted upon, however, the project requirements only mentioned HTML and CSS linters. The file provided [here](https://github.com/microverseinc/linters-config/blob/master/html-css-js/.github/workflows/linters.yml) includes a JavaScript Linter. 

**Please advise on this.**

Comment (keroloussamy) [reply .github/workflows/linters.yml:1]: Hello @Felix45 Yes in this project you just need linters for html and css only.

**Review by keroloussamy — APPROVED** [review]
Hi @Felix45, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️
- Add all the requirements. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Felix45** [comment]
Hi @keroloussamy 

Thank you for the prompt feedback.

**Pull request merged**